core/server/models/api-key.js

```diff
@@ -67,6 +113,29 @@ const ApiKey = ghostBookshelf.Model.extend({
                 this.set('role_id', null);
             }
         }
+    },
+    onSaved(model, attrs, options) {
+        if (this.previous('secret') !== this.get('secret')) {
+            addAction(model, 'refreshed', options);
```

Comment:
@kevinansfield Have named the event as `refreshed` for api keys as it seems to be consistent with rest of naming for regenerating keys, lmk if you think that makes sense